src/components/scatterplot/ScatterplotSubscriber.js

```diff
@@ -30,6 +32,7 @@ export default function ScatterplotSubscriber(props) {
     disableTooltip = false,
     observationsLabelOverride: observationsLabel = 'cell',
     observationsPluralLabelOverride: observationsPluralLabel = `${observationsLabel}s`,
+    numTicks = 50,
```

Comment:
Could we have something more descriptive as the name or make this part of a `layerOptions` object or the like?